hr_switzerland/__manifest__.py

```diff
@@ -39,8 +39,7 @@
         'views/hr_expense_custom.xml',
         'data/hr_config.xml'
     ],
-    'depends': [
-        'hr_expense', 'hr_attendance_extra_hours'],
+    'depends': ['hr_expense', 'hr_attendance_extra_hours', 'hr_attendance'],
```

Comment:
`hr_attendance_extra_hours` already depends on `hr_attendance`, so adding the dependence is useless. However, you can add dependence to `asterisk_click2dial`